### PR TITLE
Docs-cookiebar

### LINKED ISSUE #66: Cookiebar.js lacks documentation
This project has a pretty nifty cookie bar util whose use is not mentioned anywhere in the documentation. 
It makes me: (´。＿。｀)

### PR DESCRIPTION
**Pull request by MrCordeiro**
- Adds documentation for cookiebar.js. Closes #66
- Fixes cookiebar.js's `beforeDeclined` callback

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -51,3 +51,77 @@ without reloading page.
...
   </script>
+
+
+Asking users for cookie consent in templates
+--------------------------------------------
+
+``django-cookie-consent`` can website visitors a cookie consent message. This
+message informs users that the website uses cookies and requests their consent
+to store them. The script responsible for displaying the message is
+``cookiebar.js``. You can see an example of its usage in the testapp.
```
Comment (sergei-maertens) [inline docs/usage.rst:62]:
```suggestion
``django-cookie-consent`` can show website visitors a cookie consent message. This
message informs users that the website uses cookies and requests their consent
to store them. The script responsible for displaying the message is
``cookiebar.js``. You can see an example of its usage in the testapp.
```

File: docs/usage.rst [hunk 2]
```diff
@@ -51,3 +51,77 @@ without reloading page.
...
+ 
+In order to display the cookie consent message on your website:
+
+1. Load the ``cookiebar.js`` script in your HTML template. You can do this by
+   adding the following line to the ``<head>`` section of your template:
+
+.. code-block:: html
+  <script type="text/javascript" src={% static "cookie_consent/cookiebar.js" %}></script>
```
Comment (sergei-maertens) [inline docs/usage.rst:70]:
```suggestion
.. code-block:: html

  <script type="text/javascript" src="{% static 'cookie_consent/cookiebar.js' %}"></script>
```
I think this needs a blank line to properly display, and I'd also recommend properly quoting in the markup

File: docs/usage.rst [hunk 3]
```diff
@@ -51,3 +51,77 @@ without reloading page.
...
+  
+2. In your JavaScript code, call the showCookieBar function with the
```
Comment (sergei-maertens) [inline docs/usage.rst:72]:
```suggestion
2. In your JavaScript code, call the ``showCookieBar`` function with the
```

File: docs/usage.rst [hunk 4]
```diff
@@ -51,3 +51,77 @@ without reloading page.
...
+   appropriate options object:
+
+.. code-block:: javascript
+
+  showCookieBar({
```
Comment (sergei-maertens) [inline docs/usage.rst:77]:
```suggestion
  window.showCookieBar({
```

IMO that makes it a bit more obvious that it's registered as a global

In the future, we should make these scripts properly scoped & make it possible to include it in potential bundlers like webpack/rollup etc. via a proper import system.

File: docs/usage.rst [hunk 5]
```diff
@@ -51,3 +51,77 @@ without reloading page.
...
+* `content` (required): A string containing the HTML for your cookie consent
+    message.
+* `cookie_groups` (optional): An array of strings representing the cookie
+    consent groups. The script will only execute the scripts associated with
+    these groups when the user accepts cookies.
+* `cookie_decline` (optional): A string representing the cookie value to be set
+    when the user declines cookies.
+* `beforeDeclined` (optional): A callback function that runs before the user
+    declines cookies. If you don't want to run any callbacks, set this to
+    ``null``.
```
Comment (sergei-maertens) [inline docs/usage.rst:101]: please use double backticks for code to get proper code-formatting

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Really just some nitpicking in the wording/markup, other than that this seems good to merge and at least have this bit of functionality documented.

File: docs/usage.rst (hunk 5 above)
Comment (MrCordeiro) [inline docs/usage.rst:101]:
```suggestion
* ``content`` (required): A string containing the HTML for your cookie consent
    message.
* ``cookie_groups`` (optional): An array of strings representing the cookie
    consent groups. The script will only execute the scripts associated with
    these groups when the user accepts cookies.
* ``cookie_decline`` (optional): A string representing the cookie value to be set
    when the user declines cookies.
* ``beforeDeclined`` (optional): A callback function that runs before the user
    declines cookies. If you don't want to run any callbacks, set this to
    ``null``.
```

**Comment by MrCordeiro** [comment]
@sergei-maertens , I've applied you suggestions

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**